src/lightning/pytorch/loops/fetchers.py

```diff
@@ -42,24 +43,28 @@ def combined_loader(self) -> CombinedLoader:
 
     def setup(self, combined_loader: CombinedLoader) -> None:
         self._combined_loader = combined_loader
+        self.length = sized_len(combined_loader)
```

Comment:
I'm exploring some ideas how we could make this length respect the `limit_x_batches` settings from the loop: #18436